index.html

```diff
@@ -19,161 +19,176 @@
     <title>Bikes for Refugees</title>
   </head>
 
-  <body>
-    <div class="header">
-
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
+  <!-- Header Logo -->
 
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-
-        <button>Donate Now</button>
-
-    </div>
+  <body>
+    <header class="header">
+      <a href="#">
+        <img
+          src="/bikes-for-refugees/images/bikes-for-refugees_logo.jpg"
```

Comment:
When you checked your work in the deploy preview, what showed up here?

Is this path working?

Reply:
Hi sorry I had a issue when pushing css file to my branch